butter_cms/content_field.py

```diff
@@ -12,3 +12,13 @@ def get(self, keys=None, params=None):
             params = {}
         params['keys'] = ','.join(keys)
         return self.api_get(params=params)
+    
+    def getCollection(self, slug, fields=None, params=None):
```

Comment:
Could we rename this using snake_case, as a common convention in Python suggests?

Reply:
of course, my bad 🤦🏻‍♂️

Reply:
done ✅